docs/core/compatibility/windows-forms/6.0/tablelayoutsettings-apis-throw-invalidenumargumentexception.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Breaking change: Some TableLayoutSettings properties throw InvalidEnumArgumentException"
+description: Learn about the breaking change in .NET 6.0 where some TableLayoutSettings APIs now throw an InvalidEnumArgumentException for invalid arguments.
+ms.date: 01/18/2021
+---
+# Selected TableLayoutSettings properties throw InvalidEnumArgumentException
```

Comment:
@RussKie Should it be TableLayoutSettings or TableLayoutPanel?

Reply:
`TableLayoutSettings` is correct.
An instance of `TableLayoutSettings` is a backing object behind `TableLayoutPanel.CellBorderStyle` and `TableLayoutPanel.GrowStyle`.